dffml/util/cli/cmd.py

```diff
@@ -87,21 +89,32 @@ def add_subs(self, add_from: "CMD"):
                 except argparse.ArgumentError as error:
                     raise Exception(repr(add_from)) from error
 
```

Comment:
TODO, only call `self.add_argument` using args that don't have a `position` in this loop, append ones with a position to an array or dict, after the loop, sort it by position, and then make another loop where you add them in the correct order